### PR TITLE
Clarify deprecation notice

### LINKED ISSUE #118: Deprecated: What Now?
I notice the repo is now deprecated, I have a few questions:

- Why has this been deprecated?
- What is the recommended path forward?

I thought at some point there was a goal to merge Calico and Flannel daemons into a single Canal daemon? Did this not prove beneficial?

- Should we just follow standard Calico guides?
- You linked to Canal instructions for v2.6 but the same category is not present in v3.0 docs.
-  v3.0 references this repo for Canal instructions
- Standard "Calico policy with Calico networking" is still listed as beta in v3.0

I really appreciate all your hard work and efforts and it saddens me to see this project deprecated.
I'm certain it's not without good reason.
I know the root Calico project is fully capable and don't mind reverting to it, but it seemed Canal would be the eventual preference.
Is the Flannel aspect no longer necessary?

### PR DESCRIPTION
**Pull request by tmjd**
Fixes #118

### CODE REVIEW

File: InstallGuide.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
-# Up to date installation directions and manifests can be found in the [Calico docs](https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/canal)
+# [Canal docs](https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/canal) can now be found in the [Calico docs](https://docs.projectcalico.org/)
```
Comment (caseydavenport) [inline InstallGuide.md]: "can now be found on docs.projectcalico.org" reads a bit better to me.

Comment (tmjd) [reply InstallGuide.md]: Please see if you like what I've got now. A bit of a combination of your tweak with something suggested by @ozdanborne

File: InstallGuide.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
-# Up to date installation directions and manifests can be found in the [Calico docs](https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/canal)
+# [Canal manifests and docs](https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/canal) have moved to [docs.projectcalico.org](https://docs.projectcalico.org/)
 
+Up to date installation directions and manifests can be found
+[here](https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/canal).
```
Comment (ozdanborne) [inline InstallGuide.md]: [according to the style guide](https://github.com/projectcalico/calico/blob/master/DOC_STYLE_GUIDE.md#link-text) we shouldn't use `here` as the link text

**Comment by tmjd** [comment]
Please take another look here. Thanks

**Pull request merged**